features/step_definitions/kata.rb

```diff
@@ -227,3 +227,78 @@
     end
   end
 end
+
+Given /^Verify catalog source existence$/ do
+  project_name = 'openshift-marketplace'
+  catalog_source_name = 'redhat-operators'
+  step %Q/I switch to cluster admin pseudo user/
+  project(project_name)
+  if !catalog_source(catalog_source_name).exists?
+    logger.warn("Missing #{catalog_source_name} catalog source")
+    logger.info("Creating #{catalog_source_name}")
+    step %Q/I create "redhat-operators" catalogsource for testing/
+    sleep(120)
+    raise "Failed to create catalog source" unless catalog_source('redhat-operators').exists?
+  elsif catalog_source('redhat-operators').exists?
+    logger.info("Catalog source #{catalog_source_name} exists")
+  end
+end
+
+Given /^Install Kata operator$/ do
+  kata_ns ||= "openshift-sandboxed-containers-operator"
+  kata_operator_name = "sandboxed-containers-operator"
+  @result_operators = admin.cli_exec(:get, resource: "operators", n:kata_ns)
+  if !@result_operators[:stdout].to_s.include? kata_operator_name
+    project('openshift-marketplace')
+    step %Q/I store master major version in the :master_version clipboard/
+    logger.warn("#{kata_operator_name} is not installed")
+    logger.info("Installing #{kata_operator_name}")
+    step %Q/I switch to cluster admin pseudo user/
+    step %Q|I obtain test data file "kata/release-#{cb.master_version}/deployment.yaml"|
+    @result = user.cli_exec(:apply, f: "deployment.yaml")
+    sleep(120)
+    project(kata_ns)
```

Comment:
@valtair1891 hard-coded sleep again.

Reply:
Do we have a wrapped sleep function o something?

Reply:
@valtair1891 yes, you used something similiar ```I wait until...```

Reply:
@pruan-rht Sometimes hardcoded sleeps are exactly what you want.  Is it required that we use "I wait until"?

Reply:
@cpmeadors I wait is more dynamic in that the step watches for condition and exit immediately rather than just wait for a specific number of seconds.  The only time we should put a hard-coded wait is when we are testing a timeout settings.  Otherwise use dynamic polling is preferred.